src/Workspaces/VisualBasic/Portable/LanguageServices/VisualBasicTypeInferenceService.TypeInferrer.vb

```diff
@@ -838,7 +851,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 End If
 
                 Dim index = argumentList.Arguments.GetWithSeparators().IndexOf(previousToken)
-                Return (index + 1) \ 2
```

Comment:
In C# they first ensure they're after an open paren or a comma token, so it doesn't execute that logic in the similar case. There's probably cleanup to do here but I was trying to keep this fix minimal.